hack/kind-setup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+if [ ! -f "/usr/local/bin/kind" ]; then
+ echo "Installing KIND"
+ curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.14.0/kind-linux-amd64
+ chmod +x ./kind
+ sudo mv ./kind /usr/local/bin/kind
+else
+    echo "KIND already installed"
+fi
+
+CLUSTER_NAME=kcp
```

Comment:
Perhaps make this user-modifiable vs hard-coded?, e.g 

`CLUSTER_NAME=${CLUSTER_NAME:-kcp}`